frontend/src/components/eMIB/EditEmailActionDialog.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import LOCALIZE from "../../text_resources";
+import EditEmail from "./EditEmail";
+import EditTask from "./EditTask";
+import { Modal } from "react-bootstrap";
+
+export const ACTION_TYPE = {
+  email: "email",
+  task: "task"
+};
+
+export const EDIT_MODE = {
+  create: "create",
+  update: "update"
+};
+
+const styles = {
+  icon: {
+    float: "left",
+    marginTop: 14,
+    marginRight: 8
+  },
+  dialogHeaderText: {
+    float: "left",
+    fontSize: 20,
+    marginTop: 10,
+    marginBottom: 10
+  },
+  modalHeader: {
+    backgroundColor: "#00565E",
+    color: "white",
+    fontSize: 16,
+    fontWeight: 600
+  }
+};
+
+class EditEmailActionDialog extends Component {
+  static propTypes = {
+    showDialog: PropTypes.bool.isRequired,
+    handleClose: PropTypes.func.isRequired,
+    saveEmail: PropTypes.func.isRequired,
+    actionType: PropTypes.string.isRequired,
+    editMode: PropTypes.string.isRequired
+  };
+
+  buttonCloseAndAction = () => {
```

Comment:
Should we call that '_handleSave_' or simply '_closeButtonAction_'. Because I'm not sure to understand what '_AndAction_' means.